### PR TITLE
Documentation for libsass

### PR DESCRIPTION
**Pull request by FinalAngel**
- Adapts documentation for libsass usage
- Cleanup to follow 80-120 line rule

### CODE REVIEW

File: docs/general/basic_usage.rst [hunk 1]
```diff
@@ -2,63 +2,75 @@
...
-the same principle applies to static files::
+You can also add support for *other* Boilerplates, by adding the frontend
+files to namespaced directories in your application. This example of an
+application named `aldryn_addon` mentions only templates for sake of simplicity,
+but the same principle applies to static files::
 
     aldryn_addon
-        templates/  # the generic templates
-            aldryn_addon/
-                 base.html
```
Comment (sergii-kulakovskyi) [inline docs/general/basic_usage.rst]: ``base.html`` should be inside ``aldryn_addon/``

File: docs/codestyle/styles.rst [hunk 2]
```diff
@@ -6,53 +6,61 @@ Styles
...
 
 
 JS selectors
 ------------
 
-We use ``js-`` prefixed selectors for referencing DOM Nodes from javascript. That means that these classes have a pure
-functional pirpose and styles should **never** be applied to them. Same type of widget could be easily represented by
-completely different sets of markup.
+We use ``js-`` prefixed selectors for referencing DOM Nodes from javascript.
+That means that these classes have a pure functional pirpose and styles should
```
Comment (dhorelik) [inline docs/codestyle/styles.rst]: purpose =)

File: docs/contribution/index.rst [hunk 3]
```diff
@@ -57,21 +64,27 @@ Before starting to work on issues or features, please mind the branching model:
 - **master** is used for hotfix releases (1.1.x)
 - **develop** is used for features and issues (1.x.x)
 
-Everything that is merged to *develop* will be released within the next proper release (1.x.x). Major releases (x.x.x)
-will have their own branches but are always merged agains *develop*.
+Everything that is merged to *develop* will be released within the next proper
+release (1.x.x). Major releases (x.x.x) will have their own branches but are
+always merged agains *develop*.
```
Comment (dhorelik) [inline docs/contribution/index.rst]: `merged against`?

File: docs/general/basic_usage.rst [hunk 4]
```diff
@@ -2,63 +2,75 @@
...
-etc) support for the application, and if you like, for other Boilerplates too.
+So, although it means a little more work for you, you should **also** provide
+more generic frontend (templates, CSS etc) support for the application, and if
+you like, for other Boilerplates too.
 
-At the very least, the developers who use your application will find it easier to create templates and static file for
-it that support their own frontend conventions if they they can start with simple ones.
+At the very least, the developers who use your application will find it easier
+to create templates and static file for it that support their own frontend
+conventions if they they can start with simple ones.
```
Comment (dhorelik) [inline docs/general/basic_usage.rst]: `if they they they`

File: docs/general/basic_usage.rst (hunk 1 above)
Comment (FinalAngel) [inline docs/general/basic_usage.rst]: base should be on the templates level, aldryn_addon is just an example, but I fixed the structure

**Comment by Brankub** [comment]
LGTM :+1:

**Comment by dhorelik** [comment]
:+1:

**Pull request merged**